sphinx/tutorial/cylc/furthertopics/family-triggers.rst

```diff
@@ -0,0 +1,184 @@
+Family triggers
+===============
+
+This tutorial walks you through using family :term:`triggers <task trigger>`.
+
+Family triggers allow you to build dependencies based on sets of tasks
+sharing the same namespace, referred to as
+:ref:`families <tutorial-cylc-families>`.
+
+
+Purpose
+-------
+
+Family triggers are used to specify a condition to be met by a set of tasks
+in a particular :term:`family` that will :term:`trigger <task trigger>`
+the next task(s) in the :term:`graph`.
+
+Like regular task triggers, family triggers can depend on the success or
+failure of tasks. However, because a family can contain multiple tasks, you
+need to specify whether you are concerned with all or any of the tasks in
+that family reaching the desired state.
+
+
+Example
```

Comment:
I agree.

I had initially thought of doing this but if we do it for one we must do it for all. As many of the further topics are of the format information => practical => information => practical it might not gel well.

I will take a look at converting them all to use the ``practical`` directive towards the end.

Reply:
Good point, and in all fairness I have only looked through a few 'further topics' files so far where this change would be fairly simple but there could easily be cases where not, as you say. However if there is some easy-ish way to implement this I would encourage it as I imagine readers are much more likely to digest information via practical exercises than via a block of text to read.